src/site/content/en/blog/requestvideoframecallback-rvfc/index.md

```diff
@@ -0,0 +1,210 @@
+---
+title: |
+  The requestVideoFrameCallback (rVFC) API
```

Comment:
```suggestion
  The `requestVideoFrameCallback()` method
```

Reply:
Two reasons:
* Abbreviation: Don't invent abbreviations for methods, at least not in formal documents.
* API: The best practice is "use definite, specific, concrete language". In this context, 'method' is more specific than 'API'.